src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py

```diff
@@ -135,6 +135,7 @@ def flatten(xs):
 ref_to_datamodel_dict = {
     "abvegaoffset": dm.ABVegaOffsetModel,
     "barshadow": dm.BarshadowModel,
+    "bkg": None,
```

Comment:
The existing rmap does contain entries - the existing entries are WfssBkgModels, but soon (...) there will be a multiples dict required, as a new datamodel is expected for SossBkgModel.

Reply:
Thanks. So the `bkg` should open with `WfssBkgModel` (for now)? If so, do you suggest replacing the `None` with `dm.WfssBkgModel`?

Reply:
That seems like the right choice for now. The SOSS background reference file structure and algorithm are still under development (JP-3825), but at least this would cover the WFSS entries.

Reply:
Hopefully that strange last line in the rmap with no return value doesn't cause issues...

Reply:
Updated to use `WfssBkgModel` in https://github.com/spacetelescope/stdatamodels/pull/457/commits/cb96217577b447009bac8b3991b92d36df8de05e